app/views/angular/boot/_piwik.js

```diff
@@ -0,0 +1,13 @@
+var _paq = _paq || [];
+_paq.push(['trackPageView']);
+_paq.push(['enableLinkTracking']);
+(function() {
+  var u="//piwik.loomio.org/piwik/";
+  _paq.push(['setTrackerUrl', u+'piwik.php']);
+  _paq.push(['setSiteId', 1]);
+  var d=document, g=d.createElement('script'), s=d.getElementsByTagName('script')[0];
+  g.type='text/javascript'; g.async=true; g.defer=true; g.src=u+'piwik.js'; s.parentNode.insertBefore(g,s);
+})();
+
+// ???
+// <noscript><p><img src="//piwik.loomio.org/piwik/piwik.php?idsite=1" style="border:0;" alt="" /></p></noscript>
```

Comment:
Oh right, have to do something about this. What does this do again? Seems mighty fishy to me.